reframe/__init__.py

```diff
@@ -6,7 +6,7 @@
 import os
 import sys
 
-VERSION = '3.13.0-dev.0'
+VERSION = '4.0.0-dev.11'
```

Comment:
Because I was incrementing that every time I merged master into it. The initial idea was that the `4.x` would be more active, thus the proper versioning. Since the versions have never been tagged, I'm fine to start over with 4.0.0-dev.0.